src/components/Footer.astro

```diff
@@ -0,0 +1,65 @@
+---
+const today = new Date();
+import Footer from "../components/react/Footer.jsx";
+---
+
+<Footer />
+<!-- 
```

Comment:
do you need this comment

Reply:
nope

Reply:
In fact I don't even need the file, I can use Footer.jsx directly so I'll do that :D